### PR TITLE
Amend the constitution to use gender-neutral pronouns

### PR DESCRIPTION
**Pull request by alexwlchan**
This patch modifies the constitution to use gender-neutral pronouns “they/them”, rather than gendered pronouns “he/him” and “she/her”. I think I’ve got them all, but I haven’t been through with a fine-toothed comb to check.

(§28 explains how an amendment to the constitution is accepted, but doesn’t give a mechanism for proposing one. A pull request felt like a good starting point.)

### CODE REVIEW

File: constitution/12-charity_trustees.tex [hunk 1]
```diff
@@ -6,13 +6,13 @@ \section{Charity Trustees}
     The charity trustees shall manage the affairs of \shortname{} and may for that purpose exercise all the powers of \shortname{}. It is the duty of each charity trustee:
 
         \subsubsection{}
-        to exercise his or her powers and to perform his or her functions as a trustee of \shortname{} in the way he or she decides in good faith would be most likely to further the purposes of \shortname{}; and
+        to exercise teir powers and to perform their functions as a trustee of \shortname{} in the way they decide in good faith would be most likely to further the purposes of \shortname{}; and
```
Comment (meshy) [inline constitution/12-charity_trustees.tex]: Typo, `teir`, here.

Comment (alexwlchan) [reply constitution/12-charity_trustees.tex]: Oops, fixed, thanks!

**Comment by mxsasha** [comment]
I've noticed another in 9.1.1, but can't easily amend this PR.

**Comment by alexwlchan** [comment]
> @mxsasha I've noticed another in 9.1.1, but can't easily amend this PR.

Oops! Just pushed a patch which fixes it – thanks for letting me know, and do shout if you see any more.

**Comment by alexwlchan** [comment]
Just to note some conversations I’ve had with @meatballs at PyCon UK this week, and elsewhere – there’s general agreement and support for this change, but because the UKPA is now a legal entity, there are legal shenanigans* associated with changing the constitution that aren’t as easy as just merging a pull request. People well-versed in such legal matters (i.e., not me) are working out what the next steps are.

Rest assured this change has not simply been forgotten, ignored or rejected.

\* Technical term.

**Comment by meatballs** [comment]
I now have clarification of how we make this (and other) change(s) to our constitution. Only in very special circumstances do we need the permission of the Charity Commission first. In all other cases, we simply need to comply with Section 28 of our constitution which means passing a resolution at an AGM.

Sorry I didn't get this sorted in time for this year's meeting. We'll get in on the agenda for 2018.

File: constitution/15-retirement_and_removal_of_charity_trustees.tex [hunk 2]
```diff
@@ -2,10 +2,10 @@
 \section{Retirement and Removal of Charity Trustees}\label{sec:retirement_removal}
 
     \subsection{}\label{sec:retirement}
-    A charity trustee ceases to hold office if he or she:
+    A charity trustee ceases to hold office if they:
     \begin{enumerate}
         \item retires by notifying \shortname{} in writing (but only if enough charity trustees will remain in office when the notice of resignation takes effect to form a quorum for meetings);
```
Comment (alexwlchan) [inline constitution/15-retirement_and_removal_of_charity_trustees.tex:7]: “if they retires”

I need to review this more carefully before putting it to the members. Not much time before the 2018 AGM! 😉

**Comment by meatballs** [comment]
Sorry, I forgot this PR was here and I merged the changes from your branch directly.

**Comment by meatballs** [comment]
But it appears I made a mess of it. Re-opening...

**Pull request merged**